src/Ledger/Utxo/Properties.lagda

```diff
@@ -192,22 +218,22 @@ module DepositHelpers
 
   dep-ref : tot ≡ 0 → uDep + ref ≡ dep
   dep-ref tot≡0 = ℤ.+-injective $ begin
-    ℤ.+_ (uDep + ref)          ≡⟨ ℤ.pos-+ uDep ref ⟩
-    ℤ.+_ uDep ℤ.+ (ref ⊖ 0)    ≡˘⟨ cong! tot≡0 ⟩
-    ℤ.+_ uDep ℤ.+ (ref ⊖ tot)  ≡⟨ cong (ℤ._+_ (ℤ.+ uDep)) (ℤ.⊖-swap ref tot) ⟩
-    ℤ.+_ uDep ℤ.- (tot ⊖ ref)  ≡˘⟨ cong! deposits-change' ⟩
-    ℤ.+_ uDep ℤ.- Δdep         ≡˘⟨ cong (ℤ._+_ (ℤ.+ uDep)) (ℤ.⊖-swap dep uDep) ⟩
-    ℤ.+_ uDep ℤ.+ (dep ⊖ uDep) ≡⟨ ℤ.distribʳ-⊖-+-pos uDep dep uDep ⟩
-    (uDep + dep) ⊖ uDep        ≡⟨ cong (_⊖ uDep) (+-comm uDep dep) ⟩
-    (dep + uDep) ⊖ uDep        ≡˘⟨ ℤ.distribʳ-⊖-+-pos dep uDep uDep ⟩
-    ℤ.+_ dep ℤ.+ (uDep ⊖ uDep) ≡⟨ cong! (ℤ.n⊖n≡0 uDep) ⟩
-    ℤ.+_ dep ℤ.+ ℤ.0ℤ          ≡⟨ ℤ.+-identityʳ _ ⟩
-    ℤ.+_ dep ∎
+    (uDep + ref)⁺           ≡⟨ ℤ.pos-+ uDep ref ⟩
+    uDep ⁺ + (ref ⊖ 0)      ≡˘⟨ cong (λ x → uDep ⁺ + (ref ⊖ x)) tot≡0 ⟩
+    uDep ⁺ + (ref ⊖ tot)    ≡⟨ cong (_+_ (uDep ⁺)) (ℤ.⊖-swap ref tot) ⟩
+    uDep ⁺ - (tot ⊖ ref)    ≡˘⟨ cong! deposits-change' ⟩
+    uDep ⁺ - Δdep           ≡˘⟨ cong (uDep ⁺ +_) (ℤ.⊖-swap dep uDep) ⟩
+    uDep ⁺ + (dep ⊖ uDep)   ≡⟨ ℤ.distribʳ-⊖-+-pos uDep dep uDep ⟩
+    uDep + dep ⊖ uDep       ≡⟨ cong (_⊖ uDep) (+-comm uDep dep) ⟩
+    dep + uDep ⊖ uDep       ≡˘⟨ ℤ.distribʳ-⊖-+-pos dep uDep uDep ⟩
+    dep ⁺  + (uDep ⊖ uDep)  ≡⟨ cong (dep ⁺ +_) (ℤ.n⊖n≡0 uDep) ⟩
+    dep ⁺ + 0ℤ              ≡⟨ ℤ.+-identityʳ _ ⟩
+    dep ⁺                   ∎
```

Comment:
@jmchapman re. your comment about `ℤ.+ n` -> `n ⁺` being a "step too far" ...do you feel the same here?  I think the superscript + notation for the positive part makes this proof look much nicer.  Please let me know if you prefer to revert to the blackboard-bold Z in this case as well.